mlrun/db/httpdb.py

```diff
@@ -830,43 +832,61 @@ def list_projects(
         }
 
         error_message = f"Failed listing projects, query: {params}"
-        resp = self.api_call("GET", "projects", error_message, params=params)
-        return schemas.ProjectsOutput(**resp.json())
+        response = self.api_call("GET", "projects", error_message, params=params)
+        return schemas.ProjectsOutput(**response.json())
```

Comment:
This may raise some concerns by @yaronha, since you're passing back to the client the API model object. In the project case it's no biggie since project is a pretty limited object, but in general there are objections there (my PR pending review deals with the same issue for feature-sets)

Reply:
Let's leave it like that for this PR, I'll look on your PR, and we'll have the discussion on what we want (generally) there, and if needed I'll align the httpdb projects API later

Reply:
We do need to see how it aligns w SDK project class

Reply:
Indeed, para para, not in this PR